packages/server/src/utils/array.ts

```diff
@@ -1,8 +1,12 @@
 // eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
 export function unique(array: any[]): any[] {
-	return array.filter((elem, index, self) => {
-		return index === self.indexOf(elem);
-	});
+	if (array.length < 100) { // O(n^2) branch, but doesn't require creating a Set
+		return array.filter((elem, index, self) => {
+			return index === self.indexOf(elem);
+		});
+	} else { // Faster than O(n^2)
+		return Array.from(new Set(array));
+	}
```

Comment:
Alternatively, removing the `.filter` branch entirely:
```suggestion
	return Array.from(new Set(array));
```